Lib/traceback.py

```diff
@@ -1106,6 +1106,11 @@ def __init__(self, exc_type, exc_value, exc_traceback, *, limit=None,
             suggestion = _compute_suggestion_error(exc_value, exc_traceback, wrong_name)
             if suggestion:
                 self._str += f". Did you mean: '{suggestion}'?"
+        elif exc_type and issubclass(exc_type, ModuleNotFoundError) and \
+                sys.flags.no_site and \
+                getattr(exc_value, "name", None) not in sys.builtin_module_names:
```

Comment:
This feature should use `stdlib_module_names`, not *builtin* module names.
```suggestion
                getattr(exc_value, "name", None) not in sys.stdlib_module_names:
```

(It could be tested by importing a non-existent stdlib module, like `msvcrt` on *nix or `grp` on Windows. The note should *not* be added for those -- you can't solve a "wrong platform" issue by adding the site-packages directory.)

Reply:
I implemented the feedback, and it case stdlib the message it's not thrown :)